src/style.css

```diff
@@ -0,0 +1,69 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./Avenir-Book.ttf");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.search-bar {
+  font-family: inherit;
+  border: 1px solid #e1e7ed;
+  width: 100%;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-weight: bold;
+}
+
+.bigger-bar {
+  font-size: 16px;
+  height: 70px;
+  padding-left: 62px;
+}
+
+.smaller-bar {
+  font-size: 14px;
+  height: 42px;
+  padding-left: 33px;
+}
+
+.big-dandruff {
+  left: 26px;
+  top: 25px;
+  height: 19px;
+  width: 19px;
+}
+
+.small-dandruff {
+  left: 13px;
+  top: 15px;
+  height: 11px;
+  width: 11px;
+}
+
+.icon {
+  position: absolute;
+}
```

Comment:
position is a property, which is good to have separately for each element, no need to move it to general class